detox/test/e2e/19.crash-handling.test.js

```diff
@@ -51,8 +53,10 @@ describe('Crash Handling', () => {
 
     if (device.getPlatform() === 'android') {
       jestExpect(error.stack).toContain('\tat java.lang.Thread.run');
-    } else {
+    } else if (!isRNNewArch) {
```

Comment:
@asafkorem I think we need to revisit and decide:
1. The test is `@legacy` so I think `iSRNNewArch` is always false (?)
2. Should we backlog a tech debt?

Reply:
It's a technical debt or a discussion that should be opened (bug / feature). It's a non-deterministic behaviour because sometimes detox catches the error on-launch and sometimes it will be thrown in the next action (post-launch).

Reply:
https://github.com/wix/Detox/pull/4711